diesel_cli/tests/setup.rs

```diff
@@ -32,6 +32,16 @@ fn setup_creates_migrations_directory() {
 
 #[test]
 #[cfg(feature="postgres")]
+fn setup_initial_migration_returns_nothing_to_console() {
+    let p = project("setup_intial_migration_returns_nothing_to_console").build();
+
+    let result = p.command("setup").run();
+
+    assert!(!result.stdout().contains("00000000000000"));
```

Comment:
Can we change the assertion to make sure it doesn't contain `Running migration` instead?